client/src/store/cropper/actions.ts

```diff
@@ -45,10 +29,19 @@ export const uploadAvatarError = (error: Error): { type: string, payload: Error
     payload: error,
 });
 
+const FILE_SIZE = 2000000;
+export const checkFileSize = (elemTarget: any): (dispatch: Dispatch) => void =>
+    (dispatch: Dispatch): void => {
+        if (elemTarget.files[0].size > FILE_SIZE) {
+            dispatch(showAlert('File is too big', 'danger'));
+            elemTarget.value = '';
```

Comment:
does it update component? did you check?

Reply:
everything works clear